_posts/2017-10-27-interview-with-patricia-falcao-francesca-colussi-tate.md

```diff
@@ -0,0 +1,60 @@
+
+**Where do you use MediaConch? Do you use MediaConch primarily for file validation, for local policy checking, for in-house quality control, for quality testing for vendor files?**  
+
+PF: As I said above, in my context we use MediaConch to check files that have resulted from migration. We usually have one or two migration sessions a year, and MediaConch makes comparing those files among themselves and with specifications a lot easier. Dave Rice analysed the specifications for QuickTime wrapper/V210 compression and created a profile for us that has already raised a series of questions about our processes and is making us reconsider how to transfer from tape to file.  
+
+FC: I mainly use MediaConch as a comparison and ‘problem solving’ tool to spot anomalies in exhibition format files, therefore I would say I use it both for local policy checking and in-house quality control. Our Exhibition Files are specifically produced each time we are preparing a work to be displayed and when encoding a video file for an exhibition we need to comply to the media player settings (in our case mainly Brightsign) and take in consideration the projector we are going to use. Sometimes unexpected problems occur like playback issues, glitches or simply the player being unable to read the file. The first step is always to check the file visually if it’s a video, but then a multiple set of tools is necessary to have a deeper understanding of the issues.  
+
+MediaConch is particularly useful when I prepare files for multiple channel installations that originally have the same or similar properties. For example if we use the same setting to encode all the files and some of them appear faulty I use MediaConch to create a local policy and spot the issues. Sometimes it’s challenging as the rules of dependency between equipment and files vary and Dave Rice helped us exploring the possibility to develop a specific policy to check the files against the Brightsign specs which I find a very interesting challenge.  
+
+**At what point in the archival process do you use MediaConch?**  
+
+PF: We would usually use MediaConch either when we receive new files, before preparing an AIP/Bag for storage or after a migration from Tape to file.  
+
+**Do you use MediaConch for MKV/FFV1/LPCM video files, for other video files, for non-video files, or something else?**  
+
+PF: So far we’ve used it for QuickTime files with either V210, ProRes, DV or H264 compression, but also for other video files, like Mpeg.  
```

Comment:
H264 --> H.264
Mpeg --> MPEG-1/2 Video (I guess it is that, as H.264 is created also by MPEG so it would be redundant)

(2 places)